core/src/impl/Kokkos_Core.cpp

```diff
@@ -55,6 +55,8 @@
 #include <cerrno>
 #ifndef _WIN32
 #include <unistd.h>
+#else
+#include <Windows.h>
```

Comment:
Is this needed because msvc includes it by default but msvc-cl doesn't?

Reply:
Needed because clang-cl also includes it, but pure clang++ doesn't.